code/segmentedtickerplant/stplog.q

```diff
@@ -11,26 +17,36 @@ createdld:{[name;date]
   ]
  };
 
-// Functions to generate log names in one of three modes //////////////////////////////
+// Functions to generate log names in one of five modes ///////////////////////////////
```

Comment:
Think I commented elsewhere but I think this reference to the number of functions should be removed so it doesn't need updated with each new function that is added.